docker/k8s/README.md

```diff
@@ -54,7 +54,7 @@ in SGX-SIM and SGX-HW mode.
     ```
 
     > The above command would run the 
-    > [propose_requests.sh](../../scripts/propose_requests.sh) as a Kubernetes job
+    > [propose_requests.sh](../../tools/propose_requests.sh) as a Kubernetes job
```

Comment:
May be it makes more sense to keep the `propose_requests.sh` script in K8s directory.  Please modify the https://github.com/hyperledger/avalon/blob/master/docker/k8s/propose-requests-job.yaml#L17 based on the location of this script